### PR TITLE
chore: add catalog info

### PR DESCRIPTION
**Pull request by ichintanjoshi**
## Description

- added catalog-info.yaml 
- changed github issue template file

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ichintanjoshi! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+# (Required) Acceptable Values: Component, Resource, System
+# Use `Component` unless you know how backstage works and what the other kinds mean.
+kind: Component
+metadata:
+  # (Required) Must be the name of the repo, without the owning organization.
+  name: 'registrar'
+  description: "The Registrar service allows external organizations to integrate with edX programs, providing REST APIs involving program structure, enrollment, and grading."
```
Comment (sarina) [inline catalog-info.yaml]:
```suggestion
  description: "The Registrar service allows external organizations to integrate with Open edX programs, providing REST APIs involving program structure, enrollment, and grading."
```

**Review by sarina — COMMENTED** [review]
@ichintanjoshi - looks like there's some yaml linting errors on the catalog-info. You should check those, and make sure any errors are corrected on other PRs.

**Comment by ichintanjoshi** [comment]
> @ichintanjoshi - looks like there's some yaml linting errors on the catalog-info. You should check those, and make sure any errors are corrected on other PRs.

Yes @sarina on it

**Comment by ichintanjoshi** [comment]
Updated linting @sarina

**Comment by sarina** [comment]
hey @ichintanjoshi I am waiting in #621 as well as in `#dev` in slack to get an answer on whether or not we can decrease the codecov threshold (or scrap codecov altogether), I'll merge this when I have a solution.

**Review by sarina — APPROVED** [review]
@ichintanjoshi - I can't seem to edit your branch. Could you do the following, then I will merge?

- [ ] Rebase the branch on master
- [ ] Squash to two commits (`chore: add catalog-info.yml` and `chore: enable github issues via config file`)

**Comment by mphilbrick211** [comment]
@sarina @ichintanjoshi - just following up on this.

**Comment by ichintanjoshi** [comment]
> @sarina @ichintanjoshi - just following up on this.

@sarina  Have rebased

**Comment by sarina** [comment]
@ichintanjoshi can you address this feedback from my last comment?

> Squash to two commits (chore: add catalog-info.yml and chore: enable github issues via config file)

This will require an interactive rebase, not merging master into this branch.

**Comment by sarina** [comment]
You will need to run 

`git rebase -i HEAD~8`

to get at the comments in your branch, and then use the commands in interactive rebase to move and squash commits. We follow a rebasing workflow so it is wise to get used to actually _rebasing_ rather than merging in master. I can only squash your commits down to one commit via the GitHub UI, and I want to maintain the two different changes made in this branch. https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d

Alternatively, you can close this PR and make a new one with the changes properly separated out, although I think that's more work.

**Comment by openedx-webhooks** [comment]
@ichintanjoshi Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/registrar&entry.867055334=https://github.com/openedx/registrar/pull/620&entry.1484655318=https://github.com/ichintanjoshi&entry.752974735=2024-05-29+17:50&entry.1917517419=2024-06-20+12:50&entry.2133058324=No) so we can improve your experience in the future.

**Comment by ichintanjoshi** [comment]
@sarina  closing this as I think I messed up the rebasing part, will raise a new one

**Pull request closed without merging**